src/backend/distributed/planner/multi_explain.c

```diff
@@ -124,85 +118,54 @@ MultiExplainOneQuery(Query *query, IntoClause *into, ExplainState *es,
 	}
 #endif
 
-	/* handle local queries in the same way as ExplainOneQuery */
-	if (localQuery)
-	{
-		PlannedStmt *plan = NULL;
-
-		INSTR_TIME_SET_CURRENT(planStart);
+	/* plan query, just like ExplainOneQuery does */
+	INSTR_TIME_SET_CURRENT(planStart);
 
-		/* plan the query */
-		plan = pg_plan_query(query, cursorOptions, params);
+	/* plan the query */
+	plan = pg_plan_query(query, cursorOptions, params);
 
-		INSTR_TIME_SET_CURRENT(planDuration);
-		INSTR_TIME_SUBTRACT(planDuration, planStart);
+	INSTR_TIME_SET_CURRENT(planDuration);
+	INSTR_TIME_SUBTRACT(planDuration, planStart);
```

Comment:
This only counts `pg_plan_query`'s execution time. Shouldn't distributed queries count the time needed to do that planning, as well?